external/anomalib/anomaly_classification/configs/padim/configuration.yaml

```diff
@@ -0,0 +1,180 @@
+description: Configuration for an anomaly classification task
+header: Configuration for an anomaly classification task
+type: CONFIGURABLE_PARAMETERS
+visible_in_ui: true
+
+model:
```

Comment:
I think this is a question that we need to discuss in the sync meeting. There are cases where some of the configurable parameters are model specific, but to be configured by the user. Early stopping above is an example. One of the algorithms uses early stopping, while the other one doesn't. But we want it to be configurable by the user. In this case, it doesn't make sense to have a single configuration.yaml file because one of the parameters becomes model-specific.

Reply:
I'm not sure if parameter enabling/disabling is supported now in configurable parameters. For instance, if the user chooses model as `<model-a>` then the `<model-b>` related parameters would be disabled in the UI. This would be a short solution, but as we start supporting more models, this would easily become significantly cluttered